.github/ISSUE_TEMPLATE/code-cleanup-refactoring.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Code Cleanup/Refactoring
+about: Describe this issue template's purpose here.
+title: "[REFACTORING]"
+labels: code cleanup
+assignees: ''
+
+---
+
+**In which files is refactoring needed?**
+State the path to which files need refactoring.
+
+**What refactoring is needed?**
+A brief description of what refactoring is needed.
```

Comment:
The description here is sort of a repeat of its heading which doesn't really provide additional useful information. It is a pretty self explanatory header, it could benefit from a little rewording though if you agree.

Reply:
Have changed this description, thanks for the feedback :)